sympy/physics/control/tests/test_lti.py

```diff
@@ -0,0 +1,258 @@
+from sympy import (symbols, Matrix, zeros, simplify, eye, exp,
+                   ones, cosh, sinh, Integral, expand, together)
+from sympy import Rational as R
+from sympy.physics.control.lti import StateSpaceModel, TransferFunctionModel
+from mpmath import e
+
+
+def test_StateSpaceModel_create():
+    a1, a2, a3, a4, b1, b2, c1, c2, d, s = symbols('a1:5, b1:3, c1:3, d, s')
+    ssm = StateSpaceModel(Matrix([a1]), Matrix([a2]),
+                          Matrix([a3]), Matrix([a4]))
+    assert ssm.represent[0] == Matrix([a1])
```

Comment:
What is the meaning of "represent"?

Reply:
I'll remove `represent` attribute soon. Everything will be determined from `args`.